### PR TITLE
move babel configs from package.json to webpack config to fix doc build issues

### PR DESCRIPTION
**Pull request by abmai**
Looks like all the build issues resulted from having `babel` config inside of `package.json`. I don't know why.

But removing them from `package.json` and putting into `webpack.config.js` solved all the things. Individual examples still work as well.

Also ensured modules are resolved to the working directory first before the imported file's directory with `modules`.

The doc website should build great now, simply go into `website/`, then `yarn && npm run start` and voila.

### CODE REVIEW

File: examples/controls/webpack.config.js [hunk 1]
```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```
Comment (ibgreen) [inline examples/controls/webpack.config.js]: Wouldn't it be great if there was a comment in front of each line in our webpack configs that explained why a particular configuration is needed?

Particularly in this case it could save someone else a lot of time later.

File: examples/geojson-animation/webpack.config.js [hunk 2]
```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```
Comment (ibgreen) [inline examples/geojson-animation/webpack.config.js]: Why is this needed inside an example? I thought this was only needed when one directory referred to another outside its tree. The `examples` folder is intended to contain stand-alone examples.

File: examples/controls/webpack.config.js (hunk 1 above)
Comment (abmai) [inline examples/controls/webpack.config.js]: Got it.

File: examples/geojson-animation/webpack.config.js (hunk 2 above)
Comment (abmai) [inline examples/geojson-animation/webpack.config.js]: Yeah, so this is needed because it's a safer way to load the babel configuration. The reason why the build was failing was because we had babel configuration in `package.json`. I'm not sure exactly why that was causing things to fail, but plucking it out of there and putting it here resolved the issues.

Comment (ibgreen) [inline examples/geojson-animation/webpack.config.js]: But why just for this example. Why was it not necessary for any of the others? 

How is the webpack.config of this example even involved when we build the website?
